Woodoku/src/entity/score_agent.py

```diff
@@ -1,20 +1,44 @@
-from typing import Dict
+GROUP_POINTS = 18
+STREAK_POINTS = 10
+COMBO_POINTS = 28
 
 
 class ScoreAgent:
     __score: int
+    __streaks: int
 
     def __init__(self):
-        pass
+        self.__score = 0
+        self.__streaks = 0
 
-    def calculate_winning(self, group_info: Dict[str, int]) -> None:
+    def calculate_winning(self, blocks: int, groups: int) -> None:
```

Comment:
~A new issue to accommodate this change might be needed.~

Reply:
added to #38